data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/util/KafkaConsumerSecurityConfigurer.java

```diff
@@ -247,15 +248,15 @@ public static String getBootStrapServersForMsk(final AwsIamAuthConfig awsIamAuth
         }
     }
 
-    public static void setAuthProperties(Properties properties, final KafkaSourceConfig sourceConfig, final Logger LOG) {
+    public static void setAuthProperties(Properties properties, final KafkaConsumerConfig sourceConfig, final Logger LOG) {
         final AwsConfig awsConfig = sourceConfig.getAwsConfig();
         final AuthConfig authConfig = sourceConfig.getAuthConfig();
-        final KafkaSourceConfig.EncryptionConfig encryptionConfig = sourceConfig.getEncryptionConfig();
+        final EncryptionConfig encryptionConfig = sourceConfig.getEncryptionConfig();
         final EncryptionType encryptionType = encryptionConfig.getType();
 
         credentialsProvider = DefaultCredentialsProvider.create();
 
-        String bootstrapServers = sourceConfig.getBootStrapServers();
+        String bootstrapServers = sourceConfig.getBootStrapServers().iterator().next();
```

Comment:
If getBootStrapServers() is returning a list, then this should concatenate all those here.